src/mazela-artur/Fsd.Artur.Cs2/Fsd.Artur.Cs2/ConsoleActions/Writers/EndMessage.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Fsd.Artur.Cs2.Animals;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Fsd.Artur.Cs2.ConsoleActions.Writers
+{
+    class EndMessage
```

Comment:
1. public class
2. raczje bym to nazwał inaczej. Nazwa sugeruje przechowywanie stanu, a tak na prawde ta klasa jedynie ma 1 metodę wyświetlającą listę zwierząt. Np. AnimalsPresenter z metodą Show() albo coś podobnego